src/views/advanced-search/index.html

```diff
@@ -216,7 +216,7 @@ <h1 class="govuk-heading-l">Advanced company search</h1>
 
                       <input class='govuk-input govuk-!-width-full' id='sicCodes' name='sicCodes' type='text'>
 
-                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label'>Browse SIC codes (opens in new tab)</a></p>
+                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label' data-event-id='advanced-search-start-page-sic-link'>Browse SIC codes (opens in new tab)</a></p>
```

Comment:
To fix the security issue raised in the PR, can `rel="noopener"` be added to the element please.